http/oidc/src/main/java/org/wildfly/security/http/oidc/Oidc.java

```diff
@@ -116,6 +125,35 @@ public class Oidc {
     public static final String X_REQUESTED_WITH = "X-Requested-With";
     public static final String XML_HTTP_REQUEST = "XMLHttpRequest";
 
+    /* Accepted Request Object Signing Algorithms for KeyCloak*/
+    public static final String NONE = "none";
+    public static final String RS_256 = "RS256";
+    public static final String HS_256 = "HS256";
+    public static final String HS_384 = "HS384";
+    public static final String HS_512 = "HS512";
+    public static final String ES_256 = "ES256";
+    public static final String ES_384 = "ES384";
+    public static final String ES_512 = "ES512";
+    public static final String ES_256K = "ES256K";
+    public static final String RS_384 = "RS384";
+    public static final String RS_512 = "RS512";
+    public static final String PS_256 = "PS256";
+    public static final String PS_384 = "PS384";
+    public static final String PS_512 = "PS512";
+
+    /* Accepted Request Object Encrypting Algorithms for KeyCloak*/
+    public static final String RSA_OAEP = "RSA-OAEP";
+    public static final String RSA_OAEP_256 = "RSA-OAEP-256";
+    public static final String RSA1_5 = "RSA1_5";
+
+    /* Accepted Request Object Encryption Methods for KeyCloak*/
+    public static final String A256GCM = "A256GCM";
```

Comment:
Same here, can we get these accepted values through discovery instead?

Reply:
It seems like most of these, I use for testing, and some of them I don't use at all. 
So, I can remove/move those to a test Class, like `KeycloakConfiguration`. For some of the others, I am using them in the actual implementation. So, I can replace them using `AlgorithmIdentifiers.` 
I can get them using Discovery and I think I do get them. But I don't think I use them yet.